src/plugins/plugins.js

```diff
@@ -240,6 +244,8 @@ define([
     plugins.Clock = Clock.default;
     plugins.Timer = Timer.default;
     plugins.DeviceClassifier = DeviceClassifier.default;
+    plugins.UserIndicator = UserIndicator.default;
+    plugins.ExampleUser = ExampleUser.default;
```

Comment:
I think maybe we should group our example plugins. eg:
``` JavaScript
plugins.example = {};
plugins.example.User = ExampleUser.default
```

Reply:
I'll move this!